cluster-autoscaler/core/scale_up_test.go

```diff
@@ -368,7 +370,9 @@ func simpleScaleUpTest(t *testing.T, config *scaleTestConfig) {
 		extraPods[i] = pod
 	}
 
-	result, err := ScaleUp(context, clusterState, extraPods, nodes, []*extensionsv1.DaemonSet{})
+	processors := ca_processors.TestProcessors()
```

Comment:
It would be nice if we didn't need this initialization by handling nil/empty values properly inside ScaleUp.

Reply:
I added handling for nil values. I still like creating this explicitly more than passing nil though.